gef-ui/server/controllers/submit-to-ukef/index.js

```diff
@@ -59,14 +63,23 @@ const createSubmissionToUkef = async (req, res) => {
       comments.push(commentObj);
       application.comments = comments;
     }
+
+    /**
+     * Change the application submission type to MIN from MIA,
+     * upon second submission. This is to ensure correct confirmation
+     * message is shown on the NJK template else isNotice() will return false
+     */
+    const isDealNotice = application.ukefDecisionAccepted ? true : isNotice(application.submissionType);
```

Comment:
CouldisDealNotice be extracted and have it's own test?

Reply:
Extracted to helper with its own unit test cases.